broadcastclient/broadcastclient.go

```diff
@@ -56,18 +56,22 @@ var FeedConfigDefault = FeedConfig{
 }
 
 type Config struct {
-	RequireChainId     bool                     `koanf:"require-chain-id"`
-	RequireFeedVersion bool                     `koanf:"require-feed-version"`
-	Timeout            time.Duration            `koanf:"timeout"`
-	URLs               []string                 `koanf:"url"`
-	Verifier           signature.VerifierConfig `koanf:"verify"`
+	ReconnectInitialBackoff time.Duration            `koanf:"reconnect-initial-backoff"`
+	ReconnectMaximumBackoff time.Duration            `koanf:"reconnect-maximum-backoff"`
+	RequireChainId          bool                     `koanf:"require-chain-id"`
+	RequireFeedVersion      bool                     `koanf:"require-feed-version"`
+	Timeout                 time.Duration            `koanf:"timeout"`
+	URLs                    []string                 `koanf:"url"`
+	Verifier                signature.VerifierConfig `koanf:"verify"`
 }
 
 func (c *Config) Enable() bool {
 	return len(c.URLs) > 0 && c.URLs[0] != ""
 }
 
 func ConfigAddOptions(prefix string, f *flag.FlagSet) {
+	f.Duration(prefix+".reconnect-initial-backoff", DefaultConfig.ReconnectInitialBackoff, "initial duration to wait before reconnect")
+	f.Duration(prefix+".reconnect-maximum-backoff", DefaultConfig.ReconnectInitialBackoff, "maximum duration to wait before reconnect")
```

Comment:
```suggestion
	f.Duration(prefix+".reconnect-maximum-backoff", DefaultConfig.ReconnectMaximumBackoff, "maximum duration to wait before reconnect")
```
Looks like copy pasta issue here

Reply:
ooh, good catch!